tests/robustness/main_test.go

```diff
@@ -219,3 +214,13 @@ func testResultsDirectory(t *testing.T) string {
 	require.NoError(t, err)
 	return path
 }
+
+func processEndpoints(clus *e2e.EtcdProcessCluster) [][]string {
+	endpoints := make([][]string, len(clus.Procs))
+	for _, proc := range clus.Procs {
+		for i, grpcEndpoint := range proc.EndpointsGRPC() {
+			endpoints[i] = append(endpoints[i], grpcEndpoint)
```

Comment:
`EndpointsGRPC()` returns all endpoints for process, however in this case one endpoint per process should be enough.